manifests/config.pp

```diff
@@ -24,7 +24,7 @@
   }
 
   if $email {
-    $_config = merge($config, { 'email' => $email })
+    $_config = stdlib::merge($config, { 'email' => $email })
```

Comment:
I'd prefer
```suggestion
    $_config = $config + { 'email' => $email }
```

Reply:
I was thinking about this exactly, then I don't need to touch metadata. Let me try one more time